### PR TITLE
PIMS-2776: New tab view inventory page

### PR DESCRIPTION
**Pull request by jdtoombs**
Breaking this off from my other stories as I am trying to iron a few things out

### CODE REVIEW

**Review by goemen — APPROVED** [review]
Looks good. Snapshots failing...

**Review by agalbrai — APPROVED** [review]

**Review by Fosol — APPROVED** [review]

File: frontend/src/components/Table/ViewPropertyCell.tsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ const ViewPropertyCell = (props: CellProps<IProperty, number>) => {
   const property = props.row.original;
   return (
     <Link
+      target="_blank"
```
Comment (devinleighsmith) [inline frontend/src/components/Table/ViewPropertyCell.tsx:16]: I think you also want: rel="noopener noreferrer"

Comment (devinleighsmith) [reply frontend/src/components/Table/ViewPropertyCell.tsx:16]: security related tags that help prevent browser hijacking.

Comment (jdtoombs) [reply frontend/src/components/Table/ViewPropertyCell.tsx:16]: Ahh ok will add!

**Pull request merged**